### PR TITLE
Add --embroider as an option for new and init

### PR DESCRIPTION
**Pull request by thoov**
Enables --embroider along with EMBER_CLI_EMBROIDER to default the build to embroider.

### CODE REVIEW

File: lib/commands/new.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ module.exports = Command.extend({
       type: String,
       description: 'Sets the base human language of the application via index.html',
     },
+    { name: 'embroider', type: Boolean, default: false },
```
Comment (rwjblue) [inline lib/commands/new.js:37]: Can you add a description here too?

File: lib/commands/init.js [hunk 2]
```diff
@@ -35,6 +35,7 @@ module.exports = Command.extend({
       type: String,
       description: 'Sets the base human language of the application via index.html',
     },
+    { name: 'embroider', type: Boolean, default: false },
```
Comment (rwjblue) [inline lib/commands/init.js:38]: Can you add a description for the flag?

**Review by rwjblue — CHANGES_REQUESTED** [review]
Overall 👍👍, I think we need to add a description for the flag (so the help output has some conceptual explanation of what its for).

**Comment by thoov** [comment]
@rwjblue should be good now

**Review by rwjblue — APPROVED** [review]

**Pull request merged**